babel.config.js

```diff
@@ -1,50 +1,9 @@
 const config = {
-	presets: [
-		[
-			'@babel/env',
-			{
-				useBuiltIns: 'entry',
-				corejs: 2,
-				// Exclude transforms that make all code slower, see https://github.com/facebook/create-react-app/pull/5278
-				exclude: [ 'transform-typeof-symbol' ],
-			},
-		],
-		'@babel/react',
-	],
-	plugins: [
-		'@babel/plugin-proposal-class-properties',
-		'@babel/plugin-proposal-export-default-from',
-		'@babel/plugin-proposal-export-namespace-from',
-		'@babel/plugin-syntax-dynamic-import',
-		[
-			'@babel/transform-runtime',
-			{
-				corejs: false, // we polyfill so we don't need core-js
-				helpers: true,
-				regenerator: false,
-				useESModules: false,
-			},
-		],
-	],
+	extends: require.resolve( '@automattic/calypso-build/babel.config.js' ),
 	overrides: [
 		{
 			test: './extensions/',
-			plugins: [
-				[
-					'@wordpress/import-jsx-pragma',
-					{
-						scopeVariable: 'createElement',
-						source: '@wordpress/element',
-						isDefault: false,
-					},
-				],
-				[
-					'@babel/transform-react-jsx',
-					{
-						pragma: 'createElement',
-					},
-				],
-			],
+			presets: [ require( '@automattic/calypso-build/babel/wordpress-element' ) ],
```

Comment:
Oh right, so to reply myself in https://github.com/Automattic/jetpack/pull/11949/files#r273077020 —  this works just fine without name normalization and `babel-preset-*` in the package name. 👍